Warsaw_Tram/TramViewModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  TramViewModel.swift
+//  Warsaw_Tram
+//
+//  Created by Małgorzata Dziubich on 26/06/16.
+//  Copyright © 2016 Małgorzata Dziubich. All rights reserved.
+//
+
+import UIKit
+import CoreLocation
+
+class TramViewModel {
```

Comment:
Good start but this object doesn't looks like ViewModel. I'd rather call it a `Tram`. 
`ViewModel` is a object between `ViewController` and `Model`. `ViewController` is responsible for respond on user action and update view, `Model` responsibility is to serialise data. `ViewModel` is responsible for business logic, what happened when user make a action, how to modify data, and what data should be pass to `ViewController` and present to User.